Adyen/validation.py

```diff
@@ -16,17 +16,66 @@
 actions['refund'] = ["modificationAmount", "originalReference"]
 actions['cancelOrRefund'] = ["originalReference"]
 
+payout_required_fields = {
+    'confirmThirdParty': (
+        'merchantAccount',
+        'originalReference'
+    ),
+    'declineThirdParty': (
+        'merchantAccount',
+        'originalReference'
+    ),
+    'storeDetail': (
+        'merchantAccount',
+        'bank.iban',
+        'bank.ownerName',
+        'bank.countryCode',
```

Comment:
Can you please remove the bank details from the validation?
If card details are provided, bank details are no longer mandatory.

Reply:
@Aleffio If I understand you correctly, the logic here cannot be properly expressed just by a static list of fields. I should have a condition here saying that "card.*" should exist, and if it does not, then require "bank.*" data. Is this correct?

Reply:
@nyuhuhuu that is not necessary, you can just remove the 'bank' fields. If neither the bank nor the card data are provided, our platform will return the error.